density/static/js/map.js

```diff
@@ -75,6 +76,7 @@ document.addEventListener('DOMContentLoaded', function() {
 
       var percent = (totalFullness / numFloorsInBuilding); //Should be from [0, 100]
       var buildingElement = innerSvg.getElementById('parent_' + building); //e.g. parent_43
+
       if(buildingElement != null) {
         buildingElement.style.opacity = percent / 100;
       }
```

Comment:
I can't comment on the old stuff, but i think there should be a space surrounded the division sign in  `(popup.offsetWidth/2)).toString()`